apps/judicial-system/backend/src/app/modules/notification/services/caseNotification/caseNotification.service.ts

```diff
@@ -875,7 +876,9 @@ export class CaseNotificationService extends BaseNotificationService {
       isIndictmentCase(theCase.type)
         ? this.formatMessage(notifications.caseCompleted.prosecutorBody, {
             courtCaseNumber: theCase.courtCaseNumber,
-            courtName: theCase.court?.name?.replace('dómur', 'dómi'),
+            courtName: theCase.court?.name
+              ? applyDativeCaseToCourtName(theCase.court?.name)
```

Comment:
This would be the pattern for all relevant places, since `courtName` can be `undefined`. We could also do: 
`applyDativeCaseToCourtName(theCase.court?.name || ")` rather then using the ternary op 🤔 But it could also be better to just assign `courtName` with a default name on fetch to avoid needing to handling it inline everywhere...which we are currently doing in many places, or not handling it at all

Reply:
My vote is on `applyDativeCaseToCourtName(theCase.court?.name || ")` or better yet, if applyDativeCaseToCourtName could handle an empty name then `applyDativeCaseToCourtName(theCase.court?.name)`

Applying this function globally before returning the case would mean that if we were ever to use the court name without the formatting, we'd need to make a change.

Reply:
Let's go with `applyDativeCaseToCourtName(theCase.court?.name || ")` 👍